app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name, "Username" %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] When I tried to signup as a new user, the app threw an error about 1 prohibited field which I did fill in before submitting the form. I want to suggest you try to look at it and please feel free to reach out to me if it works well on your side. 
The screenshots below show the filled form and the server's output logs.

Form |
:---------:|
<img width="589" alt="Screen Shot 2022-11-10 at 8 24 50 PM" src="https://user-images.githubusercontent.com/39523312/201199763-0bdf4d25-ba88-4199-9376-2b8f549c7761.png"> |

Server Output logs |
:----------------------:|
<img width="1096" alt="Screen Shot 2022-11-10 at 8 25 17 PM" src="https://user-images.githubusercontent.com/39523312/201199856-168fca9a-ee22-4c79-8ecc-41d72cec34ad.png"> |

Reply:
@mmsesay Thank you for pointing this out :100: , however, this is the regular behaviour the app must have according to the validation plus the devise configuration. I reproduced your interaction and the error comes up because you didn't fill in the name field before sending the information for the first time, so then, when you try to send it again with the name it throws you the error. Please take this into consideration as the sign-up feature is working as required :+1:

Reply:
@mmsesay Problem solved :heavy_check_mark:

Reply:
Thanks for solving the problem @karlavdelgadof